ush/wave_tar.sh

```diff
@@ -103,12 +101,12 @@
   while [[ "${tardone}" = "no" ]]
   do
 
-    nf=$(ls | awk '/'$ID.*.$filext'/ {a++} END {print a}')
+    nf=$(find . -maxdepth 1 -type f -name "*.$filext" | wc -l)
     nbm2=$(( $nb - 2 ))
     if [[ "${nf}" -ge "${nbm2}" ]]
     then
 
-      tar -cf "${ID}.${type}.tar" ./${ID}.*.${filext}
+      tar -cf "${ID}.${type}.tar" ./*.${filext}
```

Comment:
Should double quote `filext`
```suggestion
      tar -cf "${ID}.${type}.tar" ./*."${filext}"
```